Application/EdFi.Ods.Profiles.Sample/EdFi.Ods.Profiles.Sample.csproj

```diff
@@ -30,8 +30,11 @@
     <EmbeddedResource Include="Profiles.xml" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="$(SolutionDir)..\..\Ed-Fi-ODS\Application\EdFi.Ods.Standard\EdFi.Ods.Standard.csproj" />
-    <ProjectReference Include="$(SolutionDir)..\..\Ed-Fi-ODS\Application\EdFi.Ods.Api\EdFi.Ods.Api.csproj" />
-    <ProjectReference Include="$(SolutionDir)..\..\Ed-Fi-ODS\Application\EdFi.Ods.Common\EdFi.Ods.Common.csproj" />
+    <ProjectReference Include="..\..\..\Ed-Fi-Extensions\Extensions\EdFi.Ods.Extensions.Homograph\EdFi.Ods.Extensions.Homograph.csproj" />
```

Comment:
This is creating a dependency on the other repo, this ticket should only be changes to make the profiles plugins work, like your validator change, not any changes to get the plugin built manually.

Reply:
I removed all changes to get the plugin built manually.